spring-cloud-netflix-turbine/src/main/java/org/springframework/cloud/netflix/turbine/TurbineHttpConfiguration.java

```diff
@@ -55,8 +56,24 @@ public TurbineProperties turbineProperties() {
 	}
 
 	@Bean
-	public TurbineLifecycle turbineLifecycle(InstanceDiscovery instanceDiscovery) {
-		return new TurbineLifecycle(instanceDiscovery);
+	public TurbineAggregatorProperties turbineAggregatorProperties() {
+		return new TurbineAggregatorProperties();
+	}
+
+	@Bean
```

Comment:
I would also like to add @ConditionalOnMissingBean for the beans inside this configuration to ba able to override them in user application. @ryanjbaxter what do you think?